qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -816,6 +816,8 @@ def _connect_signals(self):
         page.recentlyAudibleChanged.connect(self._delayed_recently_audible_changed)
         self._tab.url_changed.connect(self._on_url_changed)
         config.instance.changed.connect(self._on_config_changed)
+        self._silence_timer.timeout.connect(functools.partial(
+            self.recently_audible_changed.emit, False))
```

Comment:
Actually in the original method the argument (`recently_audible`) was variable, here it is hardcoded to `False`. I'm not sure if it's a suitable change but we might just be able to query the [page.recentlyAudible()](https://doc.qt.io/qt-6/qwebenginepage.html#recentlyAudible-prop) method in the time handler?

Reply:
It looks variable, but is in fact not. Lines 834-835 (the ones removed by this PR) occur within an else block that guarantees `recently_audible=False`. From my understanding, the purpose of `_silence_timer` is to mimic the 2-second delay that occurs when audio is paused described by [https://doc.qt.io/archives/qt-5.14/qwebenginepage.html#recentlyAudibleChanged](url), so to me it makes sense it's only ever called with `recently_audio=False`. But I might be misunderstanding things; if the query to `page.recentlyAudible()` seems necessary I'm not necessarily opposed to the idea.

It's also worth mentioning that upon further investigation it's possible that `_silence_timer` is no longer necessary? I'm not sure; I'm not the most well-versed in PyQt but the 2-second behavior is only described for PyQt 5.14, for 5.15 and 6 it seems to no longer be there (see [https://doc.qt.io/qt-5.15/qwebenginepage.html#recentlyAudibleChanged](url) and [https://doc.qt.io/qt-6/qwebenginepage.html#recentlyAudibleChanged](url)).

Reply:
We do want it ­— not to mimic what older Qt versions did, but because otherwise the UI gets quite "nervous" from an user's perspective, with the "audible" label showing up and disappearing again rapidly when e.g. someone talks and pauses.